task.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+# task
+#   dir1
+#   dir2
+#   dir3
+#       dir4
+
+mkdir -p ./task/dir1 ./task/dir2 ./task/dir3 ./task/dir3/dir4
```

Comment:
1. `./` не нужно указывать, и так папка будет создана рядом со скриптом
2. С флагом `-p` можно не указывать папку `task/dir3`, она и так будет создана при создании dir4
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```